gems/aws-sdk-core/spec/fixtures/credentials/mock_shared_config

```diff
@@ -278,4 +278,18 @@ role_arn = arn:aws:iam:123456789012:role/role_b
 defaults_mode = standard
 
 [profile sdk_ua_app_id]
-sdk_ua_app_id = peccy-service
\ No newline at end of file
+sdk_ua_app_id = peccy-service
+
+[profile global_endpoint_url]
+endpoint_url = https://play.min.io:9000
```

Comment:
Not sure if we care, but are we supposed to be using test URLs for real external products? It implies forever support I think, and shows up on GH search.